helm_deploy/hmpps-manage-adjudications-api/values.yaml

```diff
@@ -2,6 +2,8 @@
 generic-service:
   nameOverride: hmpps-manage-adjudications-api
 
+  serviceAccountName: hmpps-manage-adjudications-api
```

Comment:
Created with
- https://github.com/ministryofjustice/cloud-platform-environments/pull/13803
- https://github.com/ministryofjustice/cloud-platform-environments/pull/13804
- https://github.com/ministryofjustice/cloud-platform-environments/pull/13806

As the IRSA module v2 service names only need to be unique within a namespace, rather than the cluster